_docs/_latest/configure/scanner/default-rules.md

```diff
@@ -1,11 +1,13 @@
 ---
 title: Default Rules
-order: 106
+order: 302
 ---
 
 # {{ page.title }}
 
-Forseti Scanner has default rules that create a [violation]({% link _docs/latest/configure/scanner/policy-violations.md %}) when their conditions are met.
+Forseti Scanner has default rules that create a [violation]({% link _docs/latest/configure/scanner/rule-violations.md %}) when their conditions are met.
+
+---
 
 * BigQuery
```

Comment:
I think these should be headings and not unordered lists.

# Bigquery
  * Rule description.

Reply:
Also, be sure the KE rules are up to date here as they are in the code. There's also extra spacing near/around the KE lines